### ISSUE TITLE
Windows scanning support

### ISSUE THREAD

**Issue by ramizpolic** [openclarity/vmclarity#1203]
As a user, I would like to be able to scan Windows machines for potential security issues.

Issues:
- Windows SBOM analyzer
- #356 - to be verified
- Windows vulnerability scanner

**Comment by tomerse-sg** [comment]
hi!
which windows distributions does the sbom supports?
do you have an eta for vulnerability scan for windows?

**Comment by ramizpolic** [comment]
The SBOM scanning works and supports Windows XP and higher. Vulnerability scanning with SBOM data for Windows hasn't been fully tested yet, but we are committed to adding this as well. We don't have any ETA yet, but will share more details once we do. Thanks for the interest @tomerse-sg!

**Comment by tomerse-sg** [comment]
hi @ramizpolic thanks for the answer!
do you know if the vulnerabilities of windows will open source as well?

**Comment by ramizpolic** [comment]
Yes @tomerse-sg, the Windows-related logic will be part of the open-source project.

**Comment by tomerse-sg** [comment]
does syft \ trivy supports kb remediation?
